src/Integrations/dashboardIntegration.ts

```diff
@@ -117,7 +117,7 @@ function createCommonEmbeddedTrailStateProps(item: QueryMetric, dashboard: Dashb
 
   const commonProps = {
     scene: embeddedTrail,
-    title: 'Explore metrics',
+    title: 'Metrics Drilldown',
```

Comment:
I suggest keeping the casing the same:

```suggestion
    title: 'Metrics drilldown',
```